api/src/main/java/ca/bc/gov/educ/eas/api/controller/v1/AssessmentStudentController.java

```diff
@@ -27,13 +27,14 @@ public class AssessmentStudentController implements AssessmentStudentEndpoint {
   private final AssessmentStudentService studentService;
   private final AssessmentStudentValidator validator;
   private final AssessmentStudentSearchService searchService;
+  private final AssessmentStudentMapper mapper;
```

Comment:
No need to inject the mapper - just use AssessmentStudentMapper.mapper.

Reply:
Changes made.